public/js/UX.js

```diff
@@ -84,6 +84,7 @@ var help = {
     "VariableLed": "Variable Led ToolTip: Variable LED inputs an 8 bit value and glows with a proportional intensity.",
     "HexDisplay": "Hex Display ToolTip: Inputs a 4 Bit Hex number and displays it.",
     "SevenSegDisplay": "Seven Display ToolTip: Consists of 7+1 single bit inputs.",
+    "SixteenSegDisplay": "Sixteen Display ToolTip: Consists of 16+1 single bit inputs.",
```

Comment:
There is an open issue (#142) to move these strings out of this file.
You can make this string part of the element's prototype (in the element's file itself), for example:
```js
SixteenSegDisplay.prototype.tooltipText = "Consists of 16+1 single bit inputs.";
```

Reply:
You can revert the change in this file.